services/QuillLMS/engines/evidence/spec/controllers/evidence/research/gen_ai/dataset_relevant_texts_controller_spec.rb

```diff
@@ -20,6 +20,19 @@ module GenAI
           allow(user).to receive(:staff?).and_return(true)
         end
 
+        def parsed_response
+          JSON.parse(response.body)
+        end
```

Comment:
I don't think this is a huge deal, since it's just style, so take or leave, but you can also use the `let` syntax for this: `let(:parsed_response) { JSON.parse(response.body) }`.  I think that's slightly more in-line with our existing patterns, but like I said it's fine as-is.